src/dotnet.ts

```diff
@@ -10,14 +10,13 @@ import { Messages } from './ui/messages';
 
 const ListRuntimesArg = '--list-runtimes';
 const execFileAsync = promisify(execFile);
-const lstatAsync = promisify(fs.lstat);
 
 // Returns an error message or undefined.
 export async function checkSupportedDotnetVersion(): Promise<string | undefined> {
   const { path: dotnetExecutable, manual } = await getDotnetExecutablePath();
   try {
-    const stats = await lstatAsync(dotnetExecutable);
-    if(!stats.isFile() && !stats.isSymbolicLink()) {
+    const stats = await fs.promises.stat(dotnetExecutable);
+    if(!stats.isFile()) {
```

Comment:
Did the previous code break something on your machine? Can you please elaborate on this change?

I know that if you don't add symlink check, then dotnet installations in /user/bin/dotnet which is a symlink would get undetected, as raised by @prvshah51 . So I really want to understand why you write a change that would break a configuration such as for @prvshah51 , what is the problem you are really trying to solve, and see how we can find some middle ground here.

Reply:
Note that this code uses `stat` instead of `lstat`, so it works both with and without symlinks.

Reply:
Oh great ! Thanks for this finding. I did not know there was a difference between `stat` and `lstat`